pwnlib/gdb.py

```diff
@@ -721,7 +718,7 @@ def findexe():
             exe_fn = adb.proc_exe
         exe = exe_fn(pid)
 
-    if not pid and not exe:
+    if not pid and not exe and not ssh:
```

Comment:
This likely breaks cross-arch debugging with QEMU

Reply:
Why `ssh` related with cross-arch debugging? Do you mean cross-arch debugging on a remote server?